### PR TITLE
fix(dsg-1477): dropdown select bugfix

### PR DESCRIPTION
**Pull request by derekdiep42**
# Basics

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?

Fixes the issue where there are inconsistencies between selected option in display vs dropdown list. Now on init and whenever an option is typed, it will highlight the corresponding item in the dropdown list as well.

## Screenshot of the affected changes
Before:

https://github.com/Nuvalence/dsgov-web/assets/56942479/64b2e2dd-43df-4b57-be29-f6e68ff61107

After:

https://github.com/Nuvalence/dsgov-web/assets/56942479/0041ea04-90be-485e-b65a-fda6b652645e

## Checklist

- [ ] I have updated the documentation accordingly
- [x] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: libs/shared/ui/nuverial/src/lib/components/select/select.component.scss [hunk 1]
```diff
@@ -28,9 +28,6 @@
 // must be outside of the host tor style the chip properly
 ::ng-deep .nuverial-select-autocomplete-panel {
   .mdc-list-item__primary-text {
-    .nuverial-autocomplete-option-selected {
```
Comment (odeljanin) [inline libs/shared/ui/nuverial/src/lib/components/select/select.component.scss:31]: @davidyoung77 are we okay to delete this, it seems like this class does not exist in our select component/mat-autocomplete?

Comment (davidyoung77) [reply libs/shared/ui/nuverial/src/lib/components/select/select.component.scss:31]: I agree

**Review by davidyoung77 — COMMENTED** [review]
These changes all look fine, I'd like to double check the different implementations in the Agency transaction detail component before merging

**Review by davidyoung77 — APPROVED** [review]
Nice work thanks

**Pull request merged**